node/src/components/console/tasks.rs

```diff
@@ -86,6 +94,29 @@ impl Display for Session {
     }
 }
 
+pub enum TempFileSerializer {
```

Comment:
That looks like _maybe_ it could go into the `util.rs` of the `console` module?

Reply:
It's a little it too specific in my mind. `util` to me is things that, if they were more numerous, could be in a separate crate.